Makefile

```diff
@@ -1,5 +1,28 @@
 #!/usr/bin/env make
 
+# setup variables
+NAME := auth0-cli
+PKG := github.com/auth0/$(NAME)
+BUILDINFOPKG := $(PKG)/internal/build-info
+
+## setup variables for build-info
+BUILDUSER := $(shell whoami)
+BUILDTIME := $(shell date -u '+%Y-%m-%d %H:%M:%S')
+VERSION := $(shell git describe --abbrev=0)
+GITCOMMIT := $(shell git rev-parse --short HEAD)
+
+GITUNTRACKEDCHANGES := $(shell git status --porcelain --untracked-files=no)
```

Comment:
Is this in use?

Reply:
It's used in the if statement to append `-dirty` to the git commit if files in the tree were modified during build

Reply:
Oh sorry I missed that (still not used to the gh mobile app)